andela/merry-ah-backend
# #162727485 Admin rights implementation and Report actions

#### What does this PR do?
This PR implements Admin rights and Report Actions

#### Description of Task to be completed?
- Admin can view and take actions on reported Arts
- Admin can deactivate and user
- A deactivated user cannot access any protected route
- Tests for the added functions
- seeding for testing

#### How should this be manually tested?
- Report n Art  and note the User ID of the reporter, the artId and the user ID of the artist
- Sign in as an admin to close any reported article eg DELETE `http://localhost:9000/api/v1/artsreport/reportId/reporterId`
- Sign in as an Admin to warn a defaulter twice and it automatically deactivated the user on the third time eg. PUT `http://localhost:9000/api/v1/artsreport/1?reporterId=1&artId=5&defaulterId=7`
- After a user has been deactivated, try using the same user to access any protected route and see it fail 

#### Any background context you want to provide?
Admin must be logged in admin

#### What are the relevant pivotal tracker stories?
[[#162727485]](https://www.pivotaltracker.com/story/show/162727485)
